hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -430,6 +432,13 @@ public void incNumSnapshotCreateFails() {
     numSnapshotCreateFails.incr();
   }
 
+  public void incNumSnapshotLists() {
+    numSnapshotLists.incr();
+  }
+
+  public void incNumSnapshotListFails() {
+    numSnapshotListFails.incr();
+  }
```

Comment:
Not related to this jira. But this reminded me that we probably want to expose the number of active/inactive snapshots here as well at some point?

cc @neils-dev @GeorgeJahad 

I filed HDDS-7506 to track this item.